CMakeLists.txt

```diff
@@ -342,5 +342,31 @@ if(UNIX AND NOT APPLE)
   endif(FREECIV_ENABLE_RULEDIT)
 endif(UNIX AND NOT APPLE)
 
+# We grab the Libertine Font package online for non-Unix (e.g. Win32, MSYS2, and macOS)
+#  For Linux, we add a dependency to the package.
+if(NOT UNIX)
```

Comment:
Maybe move to a separate file in `cmake/` to avoid cluttering the main CMakeLists too much.

Reply:
Maybe move all the install stuff at the bottom to another file?